src/Symfony/Component/Translation/Tests/TranslatorCacheTest.php

```diff
@@ -161,6 +178,45 @@ public function testDifferentCacheFilesAreUsedForDifferentSetsOfFallbackLocales(
         $this->assertEquals('bar', $translator->trans('bar'));
     }
 
+    public function testGetCatalogueBehavesConsistently()
```

Comment:
What does it mean catalogue behaves consistently? Is it possible to alter the test name to make it clearer? You spent time understanding what these tests are doing, let's try to make it easier for future readers ;)